masking/mask_terms.py

```diff
@@ -127,38 +215,33 @@ def main():
             line = line.rstrip('\n')
 
         if args.unmask:
-            tokens = line.split('\t')
-            if len(tokens) != 3:
-                raise Exception('Need three fields (output, orig source, and masked source)!')
-
-            # output, orig_source, masked_source
-            unmasked = masker.unmask(*tokens)
-            if args.json:
-                jobj['unmasked'] = unmasked
-                print(json.dumps(jobj), flush=True)
-            else:
-                print(unmasked, flush=True)
+            if not args.json:
+                raise Exception('Unmasking requires json format')
+            
+            output = jobj['masked_output']
+            masks = jobj['masks']
+            unmasked = masker.unmask(output, masks)
+            jobj['unmasked'] = unmasked
+            print(json.dumps(jobj), flush=True)
+            
         else:
             if '\t' in line:
                 orig_source, orig_target = line.split('\t', 1)
             else:
                 orig_source = line
                 orig_target = None
 
-                if args.unmask:
-                    raise Exception()
-
-            source, target = masker.mask(orig_source, orig_target)
+            masked_source, masked_target, masks = masker.mask(orig_source, orig_target)
 
-            if target is None:
+            if orig_target is None:
                 if args.json:
-                    jobj['text'] = jobj['masked'] = source
-                    print(json.dumps(jobj, ensure_ascii=True), flush=True)
+                    jobj['masked_source'] = masked_source
```

Comment:
`masked_text` here for the JSON field name (variable name is fine)

Reply:
I can do that, though it seems a bit ambiguous to say "text" if we're going to be keeping source and MT output in the json (see previous comment for unmasking)